waypoint_saver_node:
  ros__parameters:
    use_keyboard: False

    use_joy: True
    save_button: 0
    quite_button: 1
    erace_button: 2

    auto_record: False
    waypoint_interval: 2